### DISCUSSION TITLE
Can I change how many file versions of an organization owned project are saved?

### DISCUSSION THREAD

**Discussion by Notramus** [opengisch/QField#4232] [Q&A]
My organization project is quiet small (~100MB) but since there are 10 different file version I am already hitting the storage limit on my organization. 
Is there anyway to reduce the file versions permanently? We intented to upgrade to a higher storage plan in the future, when more project are added to the organization, so manually deleting the backup versions would be too much of a hassle.

**Comment by wanderingnature** [comment]
AFAIK this is not available as an option. You have to manage versions carefully as they will use up your quota.

**Reply by Notramus** [reply]
Thanks for your feedback

**Comment by jguelat** [comment]
Being able to manage the number of versions would be indeed quite useful... The 10 versions are sometimes not enough if there are a lot of different users contributing to the project. Thus it would be for example really interesting to have a configurable number of versions per user in a project.
